src/docs/platform-architecture-reference/set-up-tcp-connectivity-on-private-cloud-openshift-platform.md

```diff
@@ -234,10 +236,10 @@ Any return code other than `0` indicates a problem.  If that's the case, inquire
 
 ## Troubleshooting
 Resolve issues by trying the following:
-* Check the Service name used in the `TransportServerClaim`. It must match the existing Service
-* Make sure that there are pods associated with that Service and that they are operating correctly
-* Confirm that the `NetworkPolicy` has been created and that its `podSelector` field matches the pods that should be reachable through the `TransportServerClaim`
-* Check the `TransportServerClaim` itself for status information
+* Check the Service name used in the `TransportServerClaim`. It must match the existing Service.
+* Make sure that there are pods associated with that Service and that they are operating correctly.
```

Comment:
As per[ guidance](https://www2.gov.bc.ca/gov/content/governments/services-for-government/service-experience-digital-delivery/web-content-development-guides/web-style-guide/writing-guide/lists#:~:text=Avoid%20bulleted%20points%20with%20more,such%20as%20for%20legal%20content.) 

 - Avoid bulleted points with more than one sentence. If you must have more than one sentence, punctuate between the sentences but not at the end
- Do not use punctuation at the end of list items, even if they're complete sentences